can_tools/bootstrap_data/covid_demographics.csv

```diff
@@ -1,49 +1,58 @@
 age,race,ethnicity,sex
```

Comment:
this adds a couple of rows for demographics in the DC scraper and sorts the rows

Reply:
SG. Part of me wonders if we could avoid having to always update this manually somehow, but as long as it's not too frequent, it's probably fine.

Reply:
I think it's good to have as validation to ensure that the demographic pairs we collect are standardized. Like if a scraper started returning new data with a pair `all,all,all,non_hispanic` we would want to group that with `all,all,all,non-hispanic` instead of creating a new entry. 

It also does need to be updated pretty infrequently (once every month or two)